### ISSUE TITLE
Diffusion Screen: make it possible to hide one type of particles

### ISSUE THREAD

**Issue by arouinfar** [phetsims/gas-properties#255]
For #30 

It's not possible to limit students to a single type of particle on the Diffusion Screen. @Nancy-Salpepi noted that she usually has students explore how a single type of particles spreads out throughout the container.

In the `settingsPanel`, we can hide individual rows of controls for variables outside the scope of the lesson's goals.
![image](https://github.com/phetsims/gas-properties/assets/8419308/a2282ec8-a309-4ecd-b92e-c3dcb3f72e40)

However, this prevents exploration of a single type of particles dispersing throughout the container, as we decided to uninsturment `numberSpinner.visibleProperty`.

An easy solution would be to re-instrument the NumberSpinners for type 2 particles. The UX would be a bit clunky though, as it would require users to hide 4 controls. It's also an imperfect solution because the Data readouts and Center of Mass icon still refer to the hidden particle type.

@pixelzoom could we create a Property that would hide everything associated with type 2 particles? Let's discuss when you return.

**Comment by pixelzoom** [comment]
For discussion, I added `diffusionScreen.view.viewProperties.numberOfParticleTypesProperty`.   It looks like this in Studio:

<img width="780" alt="screenshot_3342" src="https://github.com/phetsims/gas-properties/assets/3046552/ed292290-30d4-4345-b35c-46b34c656a6e">

Setting its value to `1` results in the UI looking like this.  Particle type 2 is hidden from the settings panel and Data accordion box, and the icon changes for the "Center of Mass" checkbox.

<img width="659" alt="screenshot_3341" src="https://github.com/phetsims/gas-properties/assets/3046552/bc2f87fc-4a4f-4455-852e-a47adb70f930">

Things that I don't like about this:
(1) The empty space in the settings panel. I've tried to change this, and it's difficult because of how the code is structured to facilitate hiding (for example) `settingsPanel.massControls`.
(2) Showing the blue particle icon 6x when there is only one particle type feels redundant. We would never design the UI like this.
